src/pydrex/diagnostics.py

```diff
@@ -289,7 +289,7 @@ def misorientation_indices(
     if ncpus is None:
         ncpus = len(os.sched_getaffinity(0)) - 1
     with Pool(processes=ncpus) as pool:
-        for i, out in enumerate(pool.imap_unordered(_run, orientation_stack)):
+        for i, out in enumerate(pool.map(_run, orientation_stack)):
```

Comment:
Just wondering if the large memory usage has anything to do with `map`. See [here](https://docs.python.org/3.11/library/multiprocessing.html#multiprocessing.pool.Pool.map).

Reply:
Oh yeah I'll try with `imap` as well, that is still ordered I think.

Reply:
Switched to `imap`.